plugins/woocommerce/includes/emails/class-wc-email-new-order.php

```diff
@@ -230,12 +230,14 @@ public function init_form_fields() {
 		 */
 		public function mobile_messaging() {
 			if ( null !== $this->object ) {
+				$domain = wp_parse_url( home_url(), PHP_URL_HOST );
 				wc_get_template(
 					'emails/email-mobile-messaging.php',
 					array(
 						'order'   => $this->object,
 						'blog_id' => class_exists( 'Jetpack_Options' ) ? Jetpack_Options::get_option( 'id' ) : null,
 						'now'     => new DateTime(),
+						'domain'  => is_string( $domain ) ? $domain : '',
```

Comment:
`wp_parse_url` uses `parse_url` function which can return many different types. In this context, we only care for a valid, not malformed domain address. We don't need partial data here. Hence checking if `$domain` is a `string`.